src/sentry/api/base.py

```diff
@@ -221,6 +221,10 @@ def paginate(self, request, on_results=None, paginator_cls=Paginator,
             results = on_results(cursor_result.results)
 
         headers = {}
+        if cursor_result.hits is not None:
+            headers['X-Hits'] = cursor_result.hits
+        if cursor_result.max_hits is not None:
+            headers['X-Max-Hits'] = cursor_result.max_hits
```

Comment:
a substructure is overkill, but this is just being defensible to changes in the Cursor result if e.g. max_hits wasnt available for some reason